lib/src/delegates/asset_picker_viewer_builder_delegate.dart

```diff
@@ -175,6 +175,18 @@ abstract class AssetPickerViewerBuilderDelegate<Asset, Path> {
     );
   }
 
+  /// Call when the viewer is calling [State.didUpdateWidget].
+  /// 当预览器调用 [State.didUpdateWidget] 时操作 [State] 和 [TickerProvider]。
+  void didUpdateViewer(
```

Comment:
Could you add `@mustCallSuper` to `initStateAndTicker`, `didUpdateViewer`, and `dispose`?